src/core/src/package_managers/AptitudePackageManager.py

```diff
@@ -288,6 +291,29 @@ def get_composite_package_identifier(self, package, package_version):
 
     def install_updates_fail_safe(self, excluded_packages):
         return
+
+    def accept_eula_for_patches(self):
+        """ Accepts eula for patches based on the config provided by customers """
+        if not os.path.exists(Constants.Paths.EULA_SETTINGS):
+            self.composite_logger.log_warning("NOT accepting EULA for any patch as no corresponding EULA Settings found on the VM")
```

Comment:
None of this is customer facing - log_debug. Almost everyone other than the limited use case here will see this warning in their results - which is not desirable at all.

Reply:
Not applicable now since the implementation has changed